drydock/plugin.py

```diff
@@ -1,11 +1,86 @@
 from glob import glob
+from typing import Any, Iterable
 import os
+import click
 import pkg_resources
 
-from tutor import hooks
+from tutor import env as tutor_env
+from tutor import serialize
+from tutor import config as tutor_config
+from tutor import hooks, types
 
 from .__about__ import __version__
 
+INIT_JOBS_SYNC_WAVE = -100
+
+def _load_jobs(tutor_conf: types.Config) -> Iterable[Any]:
+    jobs = tutor_env.render_file(tutor_conf, "k8s", "jobs.yml").strip()
+    for manifest in serialize.load_all(jobs):
+        if manifest["kind"] == "Job":
+            yield manifest
+
+def get_init_tasks():
+    """Return the list of init tasks to run."""
+    init_tasks = list(hooks.Filters.CLI_DO_INIT_TASKS.iterate())
+    context = click.get_current_context().obj
+    tutor_conf = tutor_config.load(context.root)
+
+    # Standarize deprecated COMMANDS_INIT and COMMANDS_PRE_INIT Filter
+    pre_init_tasks = []
+    for service, init_path in hooks.Filters.COMMANDS_PRE_INIT.iterate():
+        pre_init_tasks.append((service, tutor_env.read_template_file(*init_path)))
+
+    init_tasks = pre_init_tasks + init_tasks
+
+    standarized_commands = []
+    for service, init_path in list(hooks.Filters.COMMANDS_INIT.iterate()):
+        standarized_commands.append((service, tutor_env.read_template_file(*init_path)))
+
+
+    init_tasks.extend(standarized_commands)
```

Comment:
can we just do `init_tasks.append` on each for loop and avoid the intermediate lists?